examples/pos-mainchain/config/default/genesis_assets.json

```diff
@@ -1041,6 +1041,91 @@
 				}
 			}
 		},
+		{
+			"module": "nft",
+			"data": {
+				"nftSubstore": [],
+				"supportedNFTsSubstore": []
+			},
+			"schema": {
+				"$id": "/nft/module/genesis",
+				"type": "object",
+				"required": ["nftSubstore", "supportedNFTsSubstore"],
+				"properties": {
+					"nftSubstore": {
+						"type": "array",
+						"fieldNumber": 1,
+						"items": {
+							"type": "object",
+							"required": ["nftID", "owner", "attributesArray"],
+							"properties": {
+								"nftID": {
+									"dataType": "bytes",
+									"minLength": 16,
+									"maxLength": 16,
+									"fieldNumber": 1
+								},
+								"owner": {
+									"dataType": "bytes",
+									"fieldNumber": 2
+								},
+								"attributesArray": {
+									"type": "array",
+									"fieldNumber": 3,
+									"items": {
+										"type": "object",
+										"required": ["module", "attributes"],
+										"properties": {
+											"module": {
+												"dataType": "string",
+												"minLength": 1,
+												"maxLength": 1,
+												"pattern": "^[a-zA-Z0-9]*$",
+												"fieldNumber": 1
+											},
+											"attributes": {
+												"dataType": "bytes",
+												"fieldNumber": 2
+											}
+										}
+									}
+								}
+							}
+						}
+					},
+					"supportedNFTsSubstore": {
+						"type": "array",
+						"fieldNumber": 2,
+						"items": {
+							"type": "object",
+							"required": ["chainID", "supportedCollectionIDArray"],
+							"properties": {
+								"chainID": {
+									"dataType": "bytes",
+									"fieldNumber": 1
```

Comment:
Perhaps, minLength and maxLength constraint can be added for chainID field, if this point is considered valid, a change is needed in the genesis assest schema as well in the LIP and implementation.

Reply:
updated.